packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -434,14 +436,21 @@ module.exports = function (log, config) {
     };
 
     if (featureFlags.isMjmlEnabledForUser(message.email, message.template)) {
-      // TODO: This code block needs to be replaced with rendering the mjml
-      // templates. In the meantime it will render with old method.
-      const localized = this.templates.render(
+      const { template, subject, templateValues, layout } = message;
+      let localized = await this.fluentLocalizer.localizeEmail(
+        template,
+        layout || 'fxa',
+        subject,
+        templateValues,
+        message.acceptLanguage
+      );
+      localizedEmailHtml = localized.html;
+      // pass old plaintext version of the email for now
+      localized = this.templates.render(
```

Comment:
I think we can remove the else block below and always render original emails. Then overwrite with mjml templates as needed.